### PR TITLE
[PAY-546] Add memo instruction to final onramp transfer, fix rent min problem

### PR DESCRIPTION
**Pull request by rickyrombo**
### Description

- Adds a memo instruction to the transfer transaction at the end of the OnRamp flow.
- Removes min padding in favor of calculating the min rent needed for the root Solana account

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

Tested by triggering sagas manually

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mjp-memo

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mjp-memo

File: packages/web/src/services/audius-backend/BuyAudio.ts [hunk 1]
```diff
@@ -42,6 +48,15 @@ export const getSolanaConnection = async () => {
   return libs().solanaWeb3Manager.connection as Connection
 }
 
+export const getRootAccountRentExemptionMinimum = async () => {
+  await waitForLibsInit()
+  const connection = await getSolanaConnection()
+  return await connection.getMinimumBalanceForRentExemption(
+    ROOT_ACCOUNT_SIZE,
+    'finalized'
```
Comment (piazzatron) [inline packages/web/src/services/audius-backend/BuyAudio.ts:56]: this could probably be 'processed' (maybe would be faster? not sure)

**Review by piazzatron — COMMENTED** [review]
Memo ❤️

File: packages/web/src/services/audius-backend/BuyAudio.ts (hunk 1 above)
Comment (rickyrombo) [reply packages/web/src/services/audius-backend/BuyAudio.ts:56]: good point!

File: packages/web/src/store/application/ui/buy-audio/sagas.ts [hunk 2]
```diff
@@ -77,9 +79,12 @@ const SOLANA_CLUSTER = process.env.REACT_APP_SOLANA_WEB3_CLUSTER
 const ERROR_CODE_INSUFFICIENT_FUNDS = 1 // Error code for when the swap fails due to insufficient funds in the wallet
 const ERROR_CODE_SLIPPAGE = 6000 // Error code for when the swap fails due to specified slippage being exceeded
 const SLIPPAGE = 3 // The slippage amount to allow for exchanges
-const MIN_PADDING = 0.00005 * LAMPORTS_PER_SOL // Buffer for SOL in wallet
 let _jup: Jupiter
 
+const MEMO_MESSAGES = {
+  [OnRampProvider.COINBASE]: 'In-App Coinbase $AUDIO Purchase'
```
Comment (rickyrombo) [inline packages/web/src/store/application/ui/buy-audio/sagas.ts:85]: @piazzatron does this memo look good?

Comment (piazzatron) [reply packages/web/src/store/application/ui/buy-audio/sagas.ts:85]: Might be nice to have a schema to it, like `ONRAMP:PURCHASE:COINBASE`?

**Review by piazzatron — APPROVED** [review]

**Pull request merged**